deps/imgui-glew/imgui_impl_glfw_gl3.cpp

```diff
@@ -395,7 +395,7 @@ void ImGui_ImplGlfwGL3_NewFrame()
     g_MouseWheel = 0.0f;
 
     // Hide OS mouse cursor if ImGui is drawing it
-    glfwSetInputMode(g_Window, GLFW_CURSOR, io.MouseDrawCursor ? GLFW_CURSOR_HIDDEN : GLFW_CURSOR_NORMAL);
+    //glfwSetInputMode(g_Window, GLFW_CURSOR, io.MouseDrawCursor ? GLFW_CURSOR_HIDDEN : GLFW_CURSOR_NORMAL);
```

Comment:
ah I forgot imgui wasn't a submodule, I've been looking at modifying some of the other submodule code (for sfml) and it's been weird might want to just add it into the repo like you've done with imgui this cnage makes sense though are you sure you don't want to just delete this line?